### PR TITLE
Bump EDC to 0.1.3

### LINKED ISSUE #43: Update EDC to 0.1.3
### Check for existing issues

- [X] Completed

### Describe the feature

Update EDC version to 0.1.3

### If applicable, add text / mockups / screenshots to help present your vision of the feature

_No response_

### PR DESCRIPTION
**Pull request by ndr-brt**
## Description
Upgrade EDC to 0.1.3

## Approach
I used a workaround to deal with the jsonld.js library bug:
when getting the entities out of the connector, after expansion, I also added a `_compacted` field that will contain the compacted entity, where the data can be get from.
The only nit is that it's needed to set its value to undefined before sending the entity to the connector eventually (this is needed on a negotiation's initialization for example).
I didn't found a way to set it as a "ghost" field (in java there is a "transient" keyword that does the job) so that it can be excluded from serialization, if anyone has any tip, it would be really appreciated.

Closes #43

### CODE REVIEW

**Comment by ndr-brt** [comment]
Failing because of a bug in json-ld library. I opened a PR here: https://github.com/digitalbazaar/jsonld.js/pull/527

File: src/controllers/management-controller.ts [hunk 1]
```diff
@@ -452,11 +426,22 @@ export class ManagementController {
...
+    return instance;
+  }
+
+  private async expandArray<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T[]> {
+    const expanded = await jsonld.expand(body);
+    return (expanded as Array<any>).map((element, index) => {
+      var instance = Object.assign(newInstance(), element);
+      instance._compacted = body[index];
+      return instance;
+    });
```
Comment (OlfaBensoussia) [inline src/controllers/management-controller.ts:445]: I suggest that we move this block of code to a different utilities file or class to keep this file only for containing management controller methods

**Review by OlfaBensoussia — APPROVED** [review]

File: src/entities/jsonld.ts [hunk 2]
```diff
@@ -1,25 +1,66 @@
...
+};
+
+export async function expandArray<T extends JsonLdObject>(body: any, newInstance: (() => T)): Promise<T[]> {
+  const expanded = await jsonld.expand(body);
+  return (expanded as Array<any>).map((element, index) => {
+    var instance = Object.assign(newInstance(), element);
+    instance._compacted = body[index];
+    return instance;
+  });
+}
```
Comment (fdionisi) [inline src/entities/jsonld.ts:18]: I'd personally like to have these somewhere else than  the `entities/` folder. But I also believe there'll be some refactor after v0.2.0, so as long as we keep this in mind, I'm good to land this now without adding complications

File: src/entities/jsonld.ts [hunk 3]
```diff
@@ -1,25 +1,66 @@
...
+   * Once that issue will be fixed, we can get rid of the _compacted and all
+   * its usages.
+   */
+  set _compacted(_compacted: any) {
+    this.__compacted = _compacted;
+  }
+
+  get _compacted() {
+    return this.__compacted;
+  }
```
Comment (fdionisi) [inline src/entities/jsonld.ts:36]: @ndr-brt @OlfaBensoussia could you shortly explain in this PR how the workaround works? I see we are favouring it over other namespace values, but I feel I'm missing something 😅

**Review by fdionisi — APPROVED** [review]
That was fast! I just have two notes. 

Lastly, let's bump `package.json` version too

File: src/entities/jsonld.ts (hunk 2 above)
Comment (ndr-brt) [reply src/entities/jsonld.ts:18]: same here, but I really think we'll need some re-engineering, the idea would be to reach 0.2.0, bump assets to v3 and then do some housekeeping

**Comment by ndr-brt** [comment]
@fdionisi what do you mean to bump version in package.json? I did that, but maybe we're talking about different things

**Comment by fdionisi** [comment]
> @fdionisi what do you mean with "bump package.json version"? I did that, but maybe we're talking about different things

Sorry, I just noticed I missed that. All good, thank you @ndr-brt and sorry for the confusion.

**Pull request merged**